Runtime/Tracking/Velocity/ArtificialVelocityApplier.cs

```diff
@@ -1,189 +1,55 @@
 ﻿namespace Zinnia.Tracking.Velocity
```

Comment:
Why is this being extracted? This class doesn't use an Update method to process anything it simply calls Apply. So if anything the wrapper should be the other way round no?

This should stay as is, and if you want something that is iProcessable then you just have it call the Apply method in this class on every moment?

Reply:
Notice originally when call Apply() once, it starts a coroutine to “process” the deceleration every frame (coroutine works post Update() but before LateUpdate()). So it is NOT equivalent to call Apply() in the moment we want to.

Reply:
right I see, so its the Deceleration is the thing that is being moved into the Process, I understand now